scanners/boostsecurityio/bundler-audit/module.yaml

```diff
@@ -2,7 +2,7 @@ api_version: 1.0
 
 
 id: boostsecurityio/bundler-audit
-name: BoostSecurity bundler-audit
+name: Bundler-audit
```

Comment:
Are you sure the case should change, name seems to be in lower case
https://github.com/rubysec/bundler-audit

Reply:
fixed